### DISCUSSION TITLE
Chrome插件折腾之路 | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#158] [Announcements]
https://blog.goalonez.site/blog/Chrome%E6%8F%92%E4%BB%B6%E6%8A%98%E8%85%BE%E4%B9%8B%E8%B7%AF.html 

 Goalonez Blog

**Comment by Rowanz9** [comment]
知音难觅，专门登录上来表达对作者的感谢，很难遇到软件口味这么相同的博客
